BaseFilter/sections/general_url.txt

```diff
@@ -170,6 +170,12 @@
 ! Vidstream
 /mellowpresence.com^$script
 !
+! Adsterra
+! new variant
+/^https:\/\/[-a-z]{8,}\.com\/[f-i](?=[a-z]*[0-9A-Z-])[0-9A-Za-z-]{3,}\/[0-9a-z](?=[\/0-9A-Za-z]{0,131}[-_])[-\/0-9A-Z_a-z]{119,132}$/$script,xmlhttprequest,match-case,third-party
```

Comment:
Is checking for `{0,131}` required? Isn't `{119,132}` enough?
The rule looks potentially slow.
Could you please test with 100-200 urls, not matched by it + almost matched (similar to mathed urls)?

And please add an example URL.

Reply:
> Is checking for {0,131} required?

I thought it may be potentially risky if without the lookahead to guarantee having at least one `-` or `_`. Removing this part on the regex101 link only reduces the steps from `185` to `161` (8.8% improvement)

Reply:
Added an example only for the new variant. The old variant is so common so I guess we don't need an example?

Reply:
But https://melongatesdexterity.com/gOA7/fo9PIY0iewRBpf8yBV/X4SgKypcH/ is short. I mean a URL, blocked by the regexp rule.

Reply:
I somehow pasted a wrong url, will fix.